### PR TITLE
Use prop-types as React.PropTypes is deprecated

### PR DESCRIPTION
**Pull request by maukoquiroga**
These changes are required if we want to upgrade our app's React version.

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import DocumentTitle from "react-document-title"
-import React, {PropTypes} from "react"
+import React from "react"
+import PropTypes from 'prop-types'
```
Comment (MattiSG) [inline src/components/app.jsx]: Use double quotes as in other import statements.

**Comment by MattiSG** [comment]
According to https://stackoverflow.com/questions/43289348 this is probably because some dependency is still using React.PropTypes. I feel that this PR would be complete only if the warning is removed.

**Comment by MattiSG** [comment]
I understand, what I'm telling you is that the app dependencies can't currently be upgraded 😉

**Comment by maukoquiroga** [comment]
> Not blocking, but not approving until understanding of the origin of the warning has been obtained.

Haven't found the origin yet.

**Comment by maukoquiroga** [comment]
`react-router` !

**Comment by maukoquiroga** [comment]
Oh! Ok I'll take a look.

**Comment by maukoquiroga** [comment]
`swagger-ui`!

**Comment by Anna-Livia** [comment]
@maukoquiroga FYI, there is a linter alert 
It would be good to change it before rebase & merge :)
```
/Users/octo-ango/Desktop/Anna-Livia/openfisca/originals/legislation-explorer/src/history.js
  6:3  error  Newline required at end of file but not found  eol-last
```

**Comment by maukoquiroga** [comment]
Hi @Anna-Livia, I can't find that file in the the repository.

**Pull request merged**